diff --git a/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle b/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
index d86e4f5..0b29af1 100644
--- a/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
+++ b/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
@@ -1,7 +1,13 @@
 buildscript {
     repositories {
-        jcenter { url "http://jcenter.bintray.com/" }
-        maven {url "http://repo.spring.io/plugins-release/"}
+        maven {
+            url "http://jcenter.bintray.com/"
+            allowInsecureProtocol = true
+        }
+        maven {
+            url "http://repo.spring.io/plugins-release/"
+            allowInsecureProtocol = true
+        }
         mavenCentral()
         maven {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
@@ -40,8 +46,14 @@ android {
 }
 
 repositories {
-    jcenter { url "http://jcenter.bintray.com/" }
-    maven {url "http://repo.spring.io/plugins-release/"}
+    maven {
+        url "http://jcenter.bintray.com/"
+        allowInsecureProtocol = true
+    }
+    maven {
+        url "http://repo.spring.io/plugins-release/"
+        allowInsecureProtocol = true
+    }
     mavenCentral()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
